lib/rubygems/update_suggestion.rb

```diff
@@ -62,8 +53,4 @@ def eligible_for_update?
   rescue StandardError # don't block install command on any problem
     false
   end
-
-  def ci?
```

Comment:
Any chance someone uses this? I'm thinking that even if someone does, it's better to break it and have them discover the better named CIDetector.

Reply:
oh wow, you mean externally? That didn't occur to me - I checked for internal usage, but I guess it's a public method >.< `UpdateSuggestion.ci?` doesn't seem like a very _natural_ thing to call, at least..

I can

```
def ci?
  Gem::CIDetector.ci?
end
```

if you think there's any chance though.